### ISSUE TITLE
No webgui after I reinstalled everything

### ISSUE THREAD

**Issue by helto4real** [EliasKotlyar/Xiaomi-Dafang-Hacks#645]
Hey I lack webgui after I reinstalled the camera by using the latest version. The video stream is available but not the webgui. 

```
==================================================
Starting the Dafang Hacks Custom Application Layer
==================================================
Loaded common functions
Created etc directory on sdcard
Bind mounted /system/sdcard/bin/busybox to /bin/busybox
Bind mounted /system/sdcard/root to /root
Bind mounted /system/sdcard/etc to /etc
wpa_supplicant: wpa_supplicant v2.5
random: Trying to read entropy from /dev/random
Successfully initialized wpa_supplicant
Initializing interface 'wlan0' conf '/system/sdcard/config/wpa_supplicant.conf' driver 'default' ctrl_interface 'N/A' bridge 'N/A'
Configuration file '/system/sdcard/config/wpa_supplicant.conf' -> '/system/sdcard/config/wpa_supplicant.conf'
Reading configuration file '/system/sdcard/config/wpa_supplicant.conf'
ctrl_interface='/var/run/wpa_supplicant'
ctrl_interface_group='0'
ap_scan=1
Priority group 0
   id=0 ssid='removed the correct one'
nl80211: Could not add multicast membership for vendor events: -2 (No such file or directory)
rfkill: Cannot open RFKILL control device
nl80211: RFKILL status not available
nl80211: Supported cipher 00-0f-ac:1
nl80211: Supported cipher 00-0f-ac:5
nl80211: Supported cipher 00-0f-ac:2
nl80211: Supported cipher 00-0f-ac:4
nl80211: Using driver-based off-channel TX
nl80211: Disable use_monitor with device_ap_sme since no monitor mode support detected
nl80211: interface wlan0 in phy phy0
nl80211: Set mode ifindex 2 iftype 2 (STATION)
nl80211: Subscribe to mgmt frames with non-AP handle 0xc9b268
nl80211: Register frame type=0xd0 (WLAN_FC_STYPE_ACTION) nl_handle=0xc9b268 match=06
nl80211: Register frame type=0xd0 (WLAN_FC_STYPE_ACTION) nl_handle=0xc9b268 match=0a07
nl80211: Register frame type=0xd0 (WLAN_FC_STYPE_ACTION) nl_handle=0xc9b268 match=0a11
nl80211: Register frame type=0xd0 (WLAN_FC_STYPE_ACTION) nl_handle=0xc9b268 match=1101
nl80211: Register frame type=0xd0 (WLAN_FC_STYPE_ACTION) nl_handle=0xc9b268 match=1102
nl80211: Register frame type=0xd0 (WLAN_FC_STYPE_ACTION) nl_handle=0xc9b268 match=0505
netlink: Operstate: ifindex=2 linkmode=1 (userspace-control), operstate=5 (IF_OPER_DORMANT)
nl80211: Could not re-add multicast membership for vendor events: -2 (No such file or directory)
nl80211: driver param='(null)'
Add interface wlan0 to a new radio phy0
nl80211: Regulatory information - country=00
nl80211: 2402-2472 @ 40 MHz 20 mBm
nl80211: 2457-2482 @ 40 MHz 20 mBm (no IR)
nl80211: 2474-2494 @ 20 MHz 20 mBm (no OFDM) (no IR)
nl80211: 5170-5250 @ 80 MHz 20 mBm (no IR)
nl80211: 5735-5835 @ 80 MHz 20 mBm (no IR)
nl80211: 57240-63720 @ 2160 MHz 0 mBm
nl80211: Added 802.11b mode based on 802.11g information
wlan0: Own MAC address: 34:ce:00:af:ce:ea
wpa_driver_nl80211_set_key: ifindex=2 (wlan0) alg=0 addr=(nil) key_idx=0 set_tx=0 seq_len=0 key_len=0
wpa_driver_nl80211_set_key: ifindex=2 (wlan0) alg=0 addr=(nil) key_idx=1 set_tx=0 seq_len=0 key_len=0
wpa_driver_nl80211_set_key: ifindex=2 (wlan0) alg=0 addr=(nil) key_idx=2 set_tx=0 seq_len=0 key_len=0
wpa_driver_nl80211_set_key: ifindex=2 (wlan0) alg=0 addr=(nil) key_idx=3 set_tx=0 seq_len=0 key_len=0
wlan0: RSN: flushing PMKID list in the driver
nl80211: Flush PMKIDs
wlan0: Setting scan request: 0.100000 sec
ENGINE: Loading dynamic engine
ENGINE: Loading dynamic engine
EAPOL: SUPP_PAE entering state DISCONNECTED
EAPOL: Supplicant port status: Unauthorized
nl80211: Skip set_supp_port(unauthorized) while not associated
EAPOL: KEY_RX entering state NO_KEY_RECEIVE
EAPOL: SUPP_BE entering state INITIALIZE
EAP: EAP entering state DISABLED
ctrl_interface_group=0
wlan0: Added interface wlan0
wlan0: State: DISCONNECTED -> DISCONNECTED
nl80211: Set wlan0 operstate 0->0 (DORMANT)
netlink: Operstate: ifindex=2 linkmode=-1 (no change), operstate=5 (IF_OPER_DORMANT)
Daemonize..
udhcpc: deleting routers
adding dns 192.168.1.1
Initialized gpios
Determined image sensor model as jxf22
dropbear:
bftpd:
lighttpd:
Startup finished!

```
Tried to access it both through http://ip_of_the_camera and https://ip_of_the_camera
please advice

**Comment by edwork** [comment]
If you can ssh into it, run `/system/sdcard/autoupdate.sh -f` 

About 2 hours ago I cloned the repo and setup a new installation and the webgui wouldn't load. This did the trick for me. If I get a chance later I'll poke into what the issue might have been.

**Comment by helto4real** [comment]
@edwork  that did the trick! Thanks man. There was a few files that were updated even though I cloned latest version. Weird! Don't close the issue cause there might me others that this happens to. THANKS AGAIN!

**Comment by jmtatsch** [comment]
That sounds very strange if not impossible. Can you please post your update.log in a gist?

**Comment by helto4real** [comment]
Sadly I dont have it the file was empty but when I looked at the output when I ran the script it stated several files as updated  including the webserver file

**Comment by edwork** [comment]
I think the issue might be Windows tampering with symlinks and line endings. Initially when I setup my cams I unzipped (using 7zip) the files straight from the git archive to my Fat32 SD card. I did this as previously I was unable to get the cam to boot having pre-staged 5 folders of the firmware_mod I had planned to deploy.

I think this issue can be resolved as like @jmtatsch mentioned it should be impossible for this to be a bug. I can try and submit some notes to the documentation including workarounds for Windows users.
